polyfill/lib/ecmascript.mjs

```diff
@@ -5388,15 +5376,13 @@ export function RoundTime(hour, minute, second, millisecond, microsecond, nanose
 }
 
 export function DaysUntil(earlier, later) {
-  return DifferenceISODate(
+  const epochDays1 = ISODateToEpochDays(
```

Comment:
Really minor: maybe use instead of `epochDaysEarlier` / `epochDaysLater` to mirror the parameter names?